charts/metallb/templates/podmonitor.yaml

```diff
@@ -75,6 +75,7 @@ spec:
 {{- toYaml .Values.prometheus.podMonitor.relabelings | nindent 4 }}
 {{- end }}
 ---
+{{- if and .Values.prometheus.rbacPrometheus .Values.prometheus.podMonitor.enabled }}
```

Comment:
Don't we need something similar under servicemonitor too?

Reply:
yes!